src/component/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useState } from "react";
+
+const Index = () => {
+  // state
+  const [state, setState] = useState("");
```

Comment:
```suggestion
  const [inputValue, setInputValue] = useState("");
```
Quand t'auras 30 000 state dans ton composant, tu seras content qu'ils aient des noms explicite comme inputValue 😅

Reply:
😄